packages/block-library/src/gallery/edit.js

```diff
@@ -189,13 +189,21 @@ function GalleryEdit( props ) {
 		if ( existingBlock ) {
 			return existingBlock.attributes;
 		}
+
+		let newClassName;
+		if ( image.className && image.className !== '' ) {
```

Comment:
Tiny nitpick, can we simplify this check?

```suggestion
		if ( image.className !== '' ) {
```

Reply:
I thought I had instances yesterday where it was undefined and an empty string, in which case we want to check if it is defined but not an empty string, but with a quick test today I couldn't replicate the empty string instance, so maybe it just needs to be `if ( image.className)`

Reply:
or we could just do a lodash isEmpty to be safe, which is used in lots of other places

Reply:
I must have missed some edge cases then in my testing. 

Leaving it as is would be fine and doesn't introduce another use of lodash that we could avoid.